kedro-telemetry/README.md

```diff
@@ -5,39 +5,20 @@
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Code Style: Black](https://img.shields.io/badge/code%20style-black-black.svg)](https://github.com/ambv/black)
 
-## Introduction
+## What is kedro-telemetry?
 
-To find out how Kedro's features are used, the [Kedro team](https://github.com/kedro-org/kedro) have created an opt-in Kedro-Telemetry [plugin](https://docs.kedro.org/en/stable/extend_kedro/plugins.html) to gather anonymised (before being sent across the internet) and aggregated usage analytics.
-
-The data it gathers will help drive future development of Kedro because we can understand how the product is being used.
+Kedro-Telemetry is a [plugin](https://docs.kedro.org/en/stable/extend_kedro/plugins.html)
+that gathers anonymised and aggregated usage analytics
+to help [the Kedro team](https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html)
+improve the software.
 
 ## Why is my data being collected?
 
-Telemetry data is made available to [project maintainers of the Kedro Project](https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html). The purpose of providing this information is to provide metrics to the maintainers on how Kedro is used. This data helps inform future product development because we can better understand if certain features are having traction with our users. We want to make sure that Kedro is as streamlined as possible and it is difficult to do this without usage analytics.
-
-
-## What about my personal data?
-
-The Kedro Project’s telemetry has been reviewed and approved under the [Telemetry Data Collection and Usage Policy](https://lfprojects.org/policies/telemetry-data-policy/) of LF Projects, LLC.
-
-
-## What data is being collected?
-
-We process your hashed computer username, which is pseudonymized information that indirectly relates to you personally. We collect the following project-related and environmental information, relying on your consent to do so:
+Read [our Telemetry collection documentation](https://docs.kedro.org/en/stable/configuration/telemetry.html)
+for further information on the intent of the data collection and what data is collected.
```

Comment:
This doesn't answer "why?" which is itself answered in the section above that says it's to improve Kedro. I'd rephrase the question along the lines "What data is collected?"

Reply:
Good point, amending